lib/Dialect/Seq/Transforms/LowerSeqToSV.cpp

```diff
@@ -136,114 +138,136 @@ void FirRegLower::lower() {
     return;
 
   // Lower the regs to SV regs.
-  SmallVector<std::pair<sv::RegOp, llvm::Optional<AsyncResetSignal>>> toInit;
+  SmallVector<RegLowerInfo> toInit;
   for (auto reg : llvm::make_early_inc_range(regs))
     toInit.push_back(lower(reg));
 
+  // Compute total width of random space.  Place non-chisel registers at the end
+  // of the space.  The Random space is unique to the initial block, due to
+  // verilog thread rules, so we can drop trailing random calls if they are
```

Comment:
I didn't know this, but it makes sense. This leads to a nice simplification in the RandomizeRegisterInit pass!